src/content/docs/nrql/nrql-tutorials/nrql-tutorial-advanced-dashboarding.mdx

```diff
@@ -53,7 +53,7 @@ import queriesnrql2tutorial20 from 'images/queries-nrql_screenshot-crop-tutorial
 
 We hope seeing example queries and explanations in the context of your own data helps you better understand how to transform data with more advanced concepts like rate of change and aggregation. In this tutorial, you'll build on the concepts found in [foundations of using NRQL](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-tutorial/) and [controlling your data with NRQL](/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-tutorial-control-your-data/) with more advanced concepts.
 
-In this tutorial, you'll learn how to advance your dashboarding by using faceting by case, advanced aggregation functions, the `EXTRAPOLATE` keyword, filtering aggregation functions, and overriding values. Specifically, you'll learn how to use:
+You'll learn how to advance your dashboarding by using faceting by case, advanced aggregation functions, the `EXTRAPOLATE` keyword, filtering aggregation functions, and overriding values. Specifically, you'll learn how to use:
```

Comment:
In general, I think it's good to avoid "You'll" but maybe in tutorials it's ok... It's tricky in this context

Reply:
I think the "you'll" is really good in tutorials as it is directing the user to action, we can definitely discuss in our next 1 on 1!